dev/prepare_release_issue.py

```diff
@@ -289,7 +289,7 @@ def generate_issue_content(
                     console.print(f"[red]The PR #{pr_number} could not be found[/]")
                     continue
 
-            if pr.user.login == "dependabot":
+            if pr.user.login == "dependabot[bot]":
```

Comment:
I need this to have it actually skip dependabot PRs:
![image](https://user-images.githubusercontent.com/14861206/228077709-4a04e9b1-368f-4842-b02e-3e6f6be543ed.png)